src/Application/Actions/Hooks/StripePaymentsUpdate.php

```diff
@@ -315,6 +318,33 @@ private function handleChargeRefunded(Event $event, Response $response): Respons
         return $this->respondWithData($response, $charge);
     }
 
+    private function handlePaymentIntentCancelled(Request $request, Event $event, Response $response): Response
```

Comment:
I think we should test receiving this hook. Since it was previously used to check for unsupported actions, we already have a likely-suitable simulated payload ready in `pi_canceled.json`